<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vtipousuario</class>
 <widget class="QMainWindow" name="Vtipousuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tipo de Usuario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BotonAdmin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Administrador</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonUser">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tipo de Ususario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonRegresa">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
